### ISSUE TITLE
how to run

### ISSUE THREAD

**Issue by jiuchongxiao** [motown-io/motown#53]
hello i had run the project ,but i ditn't visit throw browser by  "localhost:8080".how shoud i do?
the running message just as below:
[INFO] Classes directory /Users/feng/Desktop/ocppDemon/motown/target/classes does not exist
[INFO] Context path = /
[INFO] Tmp directory = /Users/feng/Desktop/ocppDemon/motown/target/tmp
[INFO] Web defaults = org/eclipse/jetty/webapp/webdefault.xml
[INFO] Web overrides =  none
[INFO] web.xml file = null
[INFO] Webapp directory = /Users/feng/Desktop/ocppDemon/motown/src/main/webapp
[INFO] jetty-9.0.4.v20130625
[INFO] No Transaction manager found - if your webapp requires one, please configure one.
[INFO] Started o.e.j.m.p.JettyWebAppContext@79b663b3{/,file:/Users/feng/Desktop/ocppDemon/motown/src/main/webapp,AVAILABLE}{file:/Users/feng/Desktop/ocppDemon/motown/src/main/webapp}
[INFO] Started ServerConnector@17271176{HTTP/1.1}{0.0.0.0:8080}
[INFO] Started Jetty Server

**Comment by markvandenbergh** [comment]
Looking at the log you've copy/pasted everything started ok.

Do you get an error when browsing to http://localhost:8080 after starting?

**Comment by jiuchongxiao** [comment]
yes the page only show white

**Comment by markvandenbergh** [comment]
Any error in the server logs?

Could you check if you get JavaScript errors in your browser?

Open JavaScript console in:
* Chrome: Ctrl+Shift+J (or Cmd+Option+J on a Mac) https://developer.chrome.com/devtools/docs/console
* Safari: Option-Command-C https://developer.apple.com/safari/tools/
* Firefox: Ctrl+Shift+J (or Cmd+Shift+J on a Mac). https://developer.mozilla.org/en/docs/Tools/Browser_Console

**Comment by hsp8712** [comment]
In terms of the logs, I guess you have ran the command "mvn jetty:run" in the directory /motown that will  be failed. Instead, try again in the dir /motown/samples/configuration-simple.

**Comment by karthikeyan622** [comment]
"In terms of the logs, I guess you have ran the command "mvn jetty:run" in the directory /motown that will be failed. Instead, try again in the dir /motown/samples/configuration-simple."

I have tried with this path /motown/samples/configuration-simple but still am getting some issues . Attached issue log file
[issue logfile.txt](https://github.com/motown-io/motown/files/72967/issue.logfile.txt)

**Comment by hsp8712** [comment]
Maybe you must execute "mvn install" in path /motown before "mvn jetty:run", try it again !

**Comment by karthikeyan622** [comment]
I have tried as suggested by you and am getting this issue now,

Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/websocket/websocket-client/9.0.4.v20130625/websocket-client-9.0.4.v20130625.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/org.objectweb.asm/3.1.0.v200803061910/org.objectweb.asm-3.1.0.v200803061910.jar (217 KB at 107.9 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/websocket/websocket-servlet/9.0.4.v20130625/websocket-servlet-9.0.4.v20130625.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/websocket/websocket-client/9.0.4.v20130625/websocket-client-9.0.4.v20130625.jar (33 KB at 52.3 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/jetty-jsp/9.0.4.v20130625/jetty-jsp-9.0.4.v20130625.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/websocket/websocket-api/9.0.4.v20130625/websocket-api-9.0.4.v20130625.jar (42 KB at 46.2 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/javax.servlet.jsp/2.2.0.v201112011158/javax.servlet.jsp-2.2.0.v201112011158.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/websocket/websocket-servlet/9.0.4.v20130625/websocket-servlet-9.0.4.v20130625.jar (11 KB at 18.0 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/org.apache.jasper.glassfish/2.2.2.v201112011158/org.apache.jasper.glassfish-2.2.2.v201112011158.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/jetty-jsp/9.0.4.v20130625/jetty-jsp-9.0.4.v20130625.jar (4 KB at 5.7 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/javax.servlet.jsp.jstl/1.2.0.v201105211821/javax.servlet.jsp.jstl-1.2.0.v201105211821.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/jetty-server/9.0.4.v20130625/jetty-server-9.0.4.v20130625.jar (346 KB at 66.1 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/org.apache.taglibs.standard.glassfish/1.2.0.v201112081803/org.apache.taglibs.standard.glassfish-1.2.0.v201112081803.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/javax.servlet.jsp.jstl/1.2.0.v201105211821/javax.servlet.jsp.jstl-1.2.0.v201105211821.jar (51 KB at 86.7 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/javax.el/2.2.0.v201303151357/javax.el-2.2.0.v201303151357.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/websocket/websocket-common/9.0.4.v20130625/websocket-common-9.0.4.v20130625.jar (144 KB at 62.7 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/com.sun.el/2.2.0.v201303151357/com.sun.el-2.2.0.v201303151357.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/javax.servlet.jsp/2.2.0.v201112011158/javax.servlet.jsp-2.2.0.v201112011158.jar (104 KB at 40.6 KB/sec)
Downloading: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/org.eclipse.jdt.core/3.8.2.v20130121/org.eclipse.jdt.core-3.8.2.v20130121.jar
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/com.sun.el/2.2.0.v201303151357/com.sun.el-2.2.0.v201303151357.jar (134 KB at 63.1 KB/sec)
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/javax.el/2.2.0.v201303151357/javax.el-2.2.0.v201303151357.jar (55 KB at 22.8 KB/sec)
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/org.apache.jasper.glassfish/2.2.2.v201112011158/org.apache.jasper.glassfish-2.2.2.v201112011158.jar (579 KB at 136.0 KB/sec)
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/org.apache.taglibs.standard.glassfish/1.2.0.v201112081803/org.apache.taglibs.standard.glassfish-1.2.0.v201112081803.jar (420 KB at 60.9 KB/sec)
Downloaded: http://repo.maven.apache.org/maven2/org/eclipse/jetty/orbit/org.eclipse.jdt.core/3.8.2.v20130121/org.eclipse.jdt.core-3.8.2.v20130121.jar (1724 KB at 136.6 KB/sec)
[INFO] Configuring Jetty for project: io.motown:motown-io
[INFO] webAppSourceDirectory not set. Defaulting to /home/kartyk/NetBeansProjects/motown.io/motown-develop/src/main/webapp
[INFO] Reload Mechanic: automatic
[INFO] Classes = /home/kartyk/NetBeansProjects/motown.io/motown-develop/target/classes
[INFO] Context path = /
[INFO] Tmp directory = /home/kartyk/NetBeansProjects/motown.io/motown-develop/target/tmp
[INFO] Web defaults = org/eclipse/jetty/webapp/webdefault.xml
[INFO] Web overrides =  none
[INFO] web.xml file = null
[INFO] Webapp directory = /home/kartyk/NetBeansProjects/motown.io/motown-develop/src/main/webapp
2015-12-28 12:01:27.679:INFO:oejs.Server:main: jetty-9.0.4.v20130625
2015-12-28 12:01:28.169:INFO:oejpw.PlusConfiguration:main: No Transaction manager found - if your webapp requires one, please configure one.
2015-12-28 12:01:29.805:INFO:oejsh.ContextHandler:main: Started o.e.j.m.p.JettyWebAppContext@4a4d2fb5{/,file:/home/kartyk/NetBeansProjects/motown.io/motown-develop/src/main/webapp,AVAILABLE}{file:/home/kartyk/NetBeansProjects/motown.io/motown-develop/src/main/webapp}
2015-12-28 12:01:29.880:INFO:oejs.ServerConnector:main: Started ServerConnector@68f281b5{HTTP/1.1}{0.0.0.0:8080}
[INFO] Started Jetty Server

I opened my localhost:8080 as well it just showing the glassfish server running status.

What I have to do now?

**Comment by markvandenbergh** [comment]
"[INFO] Started Jetty Server

I opened my localhost:8080 as well it just showing the glassfish server
running status.

What I have to do now?"

If you see the GlassFish server status that might be the problem. The
command you executed to run Motown starts a Jetty server not GlassFish.

Make sure there is nothing running on port 8080 and run "mvn jetty:run"
 again.

2015-12-28 8:22 GMT+01:00 karthikeyan <notifications@github.com>:

**Comment by karthikeyan622** [comment]
I killed the glassfish server process that was ran on 8080. Now I ran the mvn jetty:run again it was started jetty server but localhost:8080 is showing 404 exception in jetty. What might be the cause?

**Comment by markvandenbergh** [comment]
I cloned the Motown repository to a new directory on my system, and noticed that starting the configuration-simple sample gave errors on the CXF resources.

I've added the CXF dependencies in this commit: d46a343923ee220e51beeed14ae919e90b915dfd

After that I created a new directory on my system, and cloned the repo again... just to be sure.

Did a "mvn clean && mvn install" in the root directory. Then changed directory to samples/configuration-simple. Also a "mvn clean && mvn install" there. Then "mvn jetty:run" and it runs. On port 8089, not 8080.

```
2015-12-28 14:21:16.652:INFO:oejs.ServerConnector:main: Started ServerConnector@29f2fd5b{HTTP/1.1}{0.0.0.0:8089}
[INFO] Started Jetty Server
[INFO] Starting scanner at interval of 10 seconds.
2015-12-28 14:21:16,721 [Thread-13] INFO  o.atmosphere.cpr.AtmosphereFramework - Latest version of Atmosphere's JavaScript Client 2.2.2
2015-12-28 14:21:16,721 [Thread-13] INFO  o.atmosphere.cpr.AtmosphereFramework - 

	Current version of Atmosphere 2.1.3 
	Newest version of Atmosphere available 2.1.12
```

Could you follow these steps and check if it works?

* create a new directory on your system
* clone the Motown repository into that directory
* mvn clean && mvn install in the root of the Motown directory
* cd samples/configuration-simple/
* mvn clean && mvn install
* mvn jetty:run
* after startup is complete open http://localhost:8089 in your browser.

**Comment by markvandenbergh** [comment]
Closing inactive issues
